src/test/java/org/kohsuke/groovy/sandbox/SandboxTransformerTest.java

```diff
@@ -87,7 +87,7 @@ private static final class ShouldFail { }
 
     @FunctionalInterface
     public interface ExceptionHandler {
-        public void handleException(Exception e) throws Exception;
+        public void handleException(Throwable e) throws Exception;
```

Comment:
This is so that we handle `GroovyBugError` using `ErrorCollector` so that we can print additional diagnostics rather than immediately failing the test.